sdmetrics/reports/multi_table/quality_report.py

```diff
@@ -90,13 +91,16 @@ def generate(self, real_data, synthetic_data, metadata):
             prop_scores = []
             if prop == 'Parent Child Relationships':
                 for metric in metrics:
-                    score = np.nanmean(
-                        [
-                            table_breakdown['score'] for _, table_breakdown
-                            in self._metric_results['CardinalityShapeSimilarity'].items()
-                        ]
-                    )
-                    prop_scores.append(score)
+                    if 'score' in self._metric_results[metric.__name__]:
+                        prop_scores.append(self._metric_results[metric.__name__]['score'])
+                    else:
+                        score = np.nanmean(
+                            [
+                                table_breakdown['score'] for _, table_breakdown
```

Comment:
Should we use `table_breakdown.get('score', np.nan)` to prevent failed metrics ?

Reply:
I'll address this in a follow-up PR since we want to do some additional error handling for this scenario.